src/config/graphical-session/kde.md

```diff
@@ -0,0 +1,21 @@
+# KDE
+
+## Installation
+
+Install the `kde5` package, and optionally, the `kde5-baseapps` package.
+
+To use the "Networks" widget, enable the `dbus` and `NetworkManager` services.
+
+To start Plasma via `startx`, install the `xorg` package and call
+`startplasma-x11` from `~/.xinitrc`.
+
+## Display Manager
+
+To use a graphical display manager for logging in to KDE, enable the `sddm`
+service. Test the `sddm` service before enabling it:
```

Comment:
I'm never sure about this one. "log in to" or "log into"?

Reply:
My thought is that the infinitive is "to log in", not to "to log into", which [Wiktionary](https://en.wiktionary.org/wiki/log_in) seems to agree with ....

Reply:
https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to agrees with you :D, so we can go with that. Let's wait for the CI then I think we can merge it?